crates/biome_fs/src/fs.rs

```diff
@@ -431,8 +430,8 @@ impl Advices for ErrorKind {
     fn record(&self, visitor: &mut dyn Visit) -> io::Result<()> {
         match self {
 			ErrorKind::CantReadFile(path) => visitor.record_log(
-		LogCategory::Error,
-			&format!("Biome couldn't read the following file, maybe for permissions reasons or it doesn't exists: {}", path)
+		        LogCategory::Error,
+			    &format!("Biome can't read the following file, maybe for permissions reasons or it doesn't exist: {}", path)
```

Comment:
```suggestion
			    &format!("Biome can't read the following file, maybe for permissions reasons or it doesn't exist: '{}'.", path)
```

Reply:
For similar reasons, I was hoping each arm follows a consistent style, should we update all of them with quotes?

https://github.com/biomejs/biome/blob/de596fdd8b52b9ac60455b9a380685ae1b12b340/crates/biome_fs/src/fs.rs#L434-L447